tests/modules/flye/test.yml

```diff
@@ -0,0 +1,85 @@
+# According to the issue https://github.com/fenderglass/Flye/issues/164
+# Some fluctuations are expected because of the heuristics
+# Here we check the that test.assembly_info.txt contains at least one contig
+
+- name: flye test_flye_pacbio_raw
+  command: nextflow run ./tests/modules/flye -entry test_flye_pacbio_raw -c ./tests/config/nextflow.config  -c ./tests/modules/flye/nextflow.config -stub-run
```

Comment:
This command needs at lest 8Gb of memory for indexing ([#511](https://github.com/fenderglass/Flye/issues?q=is%3Aissue+is%3Aclosed+)). The tests pass locally. I add a stub run here as GH actions run in 7Gb VMs.